.github/workflows/live_tests.yml

```diff
@@ -25,6 +25,10 @@ on:
         required: true
       streams:
         description: Streams to include in tests
+      should_read_with_state:
+        description: Whether to run tests against the read command with state
```

Comment:
Can you tell me more about how this is applied? Does it mean "with taking state in at the start of read", or something else?